docs/create-project-shapefiles.md

```diff
@@ -0,0 +1,69 @@
+# Current approach of handling shapefiles
+
+## Custom planning area
+
+Frontend sends Shapefile which is persisted in geo database and its ID and 
+GeoJSON is returned to make it being displayed on map by frontend (not using 
+tiles)
+
+Then, when submitting whole creation, given ID is used.
+
+There is some simple mechanism that collects unused persisted planning areas.
+
+## Custom feature
+
+Frontend sends Shapefile which is persisted in geo database and attached to 
+scenario (without explicit marking it as used). Neither ID nor GeoJSON is 
+returned.
+
+## Protected area
+
+Frontend sends Shapefile which is persisted in geo database and attached to 
+project (without explicit marking it as used). Neither ID nor GeoJSON is
+returned.
```

Comment:
the upcoming changes will also modify the scenario-area relations.

Reply:
We are discussing directly show the new PA on the map in the current scenario.

Reply:
(and those are known after upload)